File: C Language/LevelOrderTreeTraversal.c
#include<stdio.h>
#include<stdlib.h>
#define MAX 5

struct node
{
    int data;
    struct node* left;
    struct node* right;
};

struct node* newNode(int data)
{
    struct node* ptr = (struct node*)malloc(sizeof(struct node));
    ptr -> data = data;
    ptr -> left = NULL;
    ptr -> right = NULL;
    return ptr;
}

struct node** createQueue(int *front, int *rear)
{
    struct node** queue = (struct node**)malloc(MAX*sizeof(struct node*));
    *front = *rear = 0;
    return queue;
}

void enQueue(struct node** queue, int *rear, struct node* newNode)
{
    queue[*rear] = newNode;
    (*rear)++;
}

struct node* deQueue(struct node** queue, int *front)
{
    (*front)++;
    return queue[*front-1];
}

//Display the Dequeue
void levelOrderTree(struct node* root)
{
    int front, rear;
    struct node** queue = createQueue(&front, &rear);
    struct node* temp = root;
    while(temp)
    {
        printf("%d ", temp -> data);
        
        if(temp -> left)
        {
            enQueue(queue, &rear, temp -> left);
        }
        
        if(temp -> right)
        {
            enQueue(queue, &rear, temp -> right);
        }
        
        temp = deQueue(queue, &front); 
    }
}

int main()
{
    struct node* root = newNode(1);
    root -> left = newNode(2);
    root -> right = newNode(3);
    root -> left -> left = newNode(4);
    root -> right -> right = newNode(5);
    
    printf("Level order traversal of binary tree is:\n");
    levelOrderTree(root);
    
    return 0;
}

File: Python/LevelOrderTreeTraversal.py
class BinaryTree:
	def __init__(self, data):
		self.data = data
		self.left = None
		self.right = None
		
	def levelOrderTraversal(self, root):
		if root is None:
			return
			
		queue = []
		
		#Enqueue Root and initialize height
		queue.append(root)

		while(len(queue)>0):
			print(queue[0].data, end=' ')
			temp = queue.pop(0)
			
			#Enqueue left child 
			if temp.left is not None:
				queue.append(temp.left)
				
			#Enqueue right child
			if temp.right is not None:
				queue.append(temp.right)
		
	"""def insertLeft(self, newNode):
		if self.left is None:
			self.left = BinaryTree(newNode)
		else:
			tree = BinaryTree(newNode)
			tree.left = self.left
			self.left = tree
		
	def insertRight(self, newNode):
		if self.right is None:
			self.right = BinaryTree(newNode)
		else:
			tree = BinaryTree(newNode)
			tree.right = self.right
			self.right = tree
		
root = BinaryTree(4)
root.insertLeft(8)
root.insertRight(12)
root.insertLeft(16)
root.insertRight(20)
root.insertLeft(24)
root.insertRight(28)"""

root = BinaryTree(1) 
root.left = BinaryTree(2) 
root.right = BinaryTree(3) 
root.left.left = BinaryTree(4) 
root.left.right = BinaryTree(5)

print('Level Order Traversal of binary tree is:')
root.levelOrderTraversal(root)